guardian/gateway
# Ab test 3 stage sign in flow

## What does this change?

This adds an AB test that will direct 50% of the audience to a sign in flow that places the marketing consents and newletters opt ins on one page, so that the sign in flow becomes a 3 stage process rather than a 4 stage process. The hypothesis is that more users will complete registration with a shorter process. 


## How to test

Add a cookie `GU_mvt_id` with value `1` to your browser and visit `https://profile.thegulocal.com/consents`. This should redirect you to `https://profile.thegulocal.com/consents/our_content`

## Code tests:

- [x] MVT even, A new social registration is redirected to the 3 step flow
- [x] MVT odd, A new social registration is redirected to the 4 step flow
- [x] MVT even, A new password registration is redirected to the 3 step flow
- [x] MVT odd, A new social registration is redirected to the 4 step flow



## How can we measure success?



## Have we considered potential risks?



## Images
Currently it looks like this:
<img width="736" alt="our_content_with_image" src="https://github.com/guardian/gateway/assets/1202622/35a94744-1151-4a49-b806-49851ad8f093">

The image used is this one: 
<img width="532" alt="supporter" src="https://github.com/guardian/gateway/assets/1202622/0c4b862d-3e50-4dd7-92af-524bf6fe40f3">

There is a link to the original design file here: https://www.figma.com/file/QjyXYCypmkedbk9gp0GJph/Global-New-Year-2022?type=design&node-id=983-20386&mode=design&t=OgVqYZ7kyu4QIOSd-0 

Would be great to confirm best practises to use here


## Accessibility
